<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameFieldUI</class>
 <widget class="QWidget" name="GameFieldUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="gameFieldContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>720</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>720</width>
       <height>720</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#gameFieldContainer{
background-color: #000000;
border: 5px solid back;
}</string>
     </property>
     <layout class="QGridLayout" name="gameFieldGrid">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="RightSidebar" native="true">
     <property name="styleSheet">
      <string notr="true">#RightSidebar{
background-color: rgba(0,0,0,160);
border: 2px solid black;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="SidebarTopWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>350</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="playerNameLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#playerNameLabel{
color: #dfdfdf;
}</string>
           </property>
           <property name="text">
            <string>J√°t√©kosn√©v</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="PaintedWidget" name="playerPortraitWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#playerPortraitWidget{
background-color: rgba(0,0,0,120);
border: 2px solid black;
border-radius: 10px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QLabel" name="mapNameLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#mapNameLabel{
color: #dfdfdf;
}</string>
        </property>
        <property name="text">
         <string>P√°lyan√©v</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#timeLabel{
color: #dfdfdf;
}</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
       <widget class="QWidget" name="navWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#navWidget{
background-color: rgba(0,0,0,120);
border: 2px solid black;
border-radius: 10px;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QPushButton" name="navLeft">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>24</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ClosedHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>ü°∏</string>
           </property>
           <property name="shortcut">
            <string>Left</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QPushButton" name="navSpace">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>26</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}

QPushButton:disabled[flat=true]{
background-color: rgba(74, 68, 68,170);
}</string>
           </property>
           <property name="text">
            <string>SPACE</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="navInteract">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>24</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>ClosedHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}

QPushButton:disabled[flat=true]{
background-color: rgba(74, 68, 68,170);
}</string>
           </property>
           <property name="text">
            <string>E</string>
           </property>
           <property name="shortcut">
            <string>E</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="navUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>ü°π</string>
           </property>
           <property name="shortcut">
            <string>Up</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="navDown">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>ü°ª</string>
           </property>
           <property name="shortcut">
            <string>Down</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="navRight">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #dfdfdf;
border: 2px solid #cdcdcd;
border-radius: 20px;
}

QPushButton:pressed[flat=true]{
background-color: rgba(89, 22, 9,200);
color :  #eef205;
border: 2px solid #eef205;
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>ü°∫</string>
           </property>
           <property name="shortcut">
            <string>Right</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">navButtons</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintedWidget</class>
   <extends>QWidget</extends>
   <header>paintedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="navButtons"/>
 </buttongroups>
</ui>
